cypress/integration/features/task/createEditAndDeleteTask.feature

```diff
@@ -9,7 +9,7 @@ Feature: To create, edit and delete tasks by the teacher.
     And I click on FAB to create new content
```

Comment:
file name can be --> createEditDeleteSubGradFinishRestoreTask.feature

Reply:
File name of feature and step.js are changed